indico/modules/events/registration/models/registration_form_fields.py

```diff
@@ -48,7 +59,11 @@ class RegistrationFormFieldData(db.Model):
         db.Integer,
         db.ForeignKey('event_registration.registration_form_items.id')
     )
-    # TODO: which data? price, limit, options, etc.
+    #: Data describing the field
+    data = db.Column(
+        JSON,
+        nullable=True
```

Comment:
it won't be NULL in any case since None will be stored as json `null`, not db-level `NULL`